src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -22,10 +22,12 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
 
     @Override
     public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
-        String name = req.getParameter("name");
-        String licenseNumber = req.getParameter("license_number");
-        Driver driver = new Driver(name, licenseNumber);
+        Driver driver = new Driver();
+        driver.setName(req.getParameter("name"));
+        driver.setLicenseNumber(req.getParameter("license_number"));
+        driver.setLogin(req.getParameter("login"));
+        driver.setPassword(req.getParameter("password"));
         driverService.create(driver);
-        resp.sendRedirect(req.getContextPath() + "/drivers/add");
+        resp.sendRedirect(req.getContextPath() + "/drivers/login");
```

Comment:
maybe you meant "/register"? But it is better to leave /drivers/add

Reply:
logic next - after registration, the user gets to the login page

Reply:
that's for redirecting user to login page after registration new driver